game/dcs/aircrafttype.py

```diff
@@ -367,7 +367,10 @@ def _each_variant_of(cls, aircraft: Type[FlyingType]) -> Iterator[AircraftType]:
             units = ImperialUnits()
         if units_data == "metric":
             units = MetricUnits()
-
+        prop_overrides = data.get("default_overrides")
+        if prop_overrides and aircraft.property_defaults:
```

Comment:
I think you might want `aircraft.property_defaults is not None`. I can't work out whether there's actually a different behavior (are there aircraft with an empty `property_defaults` dict that do have properties?)

Reply:
> (are there aircraft with an empty property_defaults dict that do have properties?)
I don't think so.

Still better to be explicit though, I'll update.

Reply:
I suppose it might also be good to raise if the yaml tries to set props but the dict is None. Saves the confusion of debugging why your props aren't having any effect.

Reply:
Added logging rather than raise exception, now a warning is logged if trying to set to non-existent prop.

Nothing validates that the values are okay, only that the props exist to be set.  I would expect [UnitProperty.default_values_for](https://github.com/dcs-liberation/dcs_liberation/blob/ff12b37431114f44e1a9094203780e1f9447fed1/game/dcs/unitproperty.py#L53) to perform some simple validation on values, but it doesn't seem to raise if e.g. a string is set.